docs/modules/general/proto.mdx

```diff
@@ -0,0 +1,335 @@
+- A team can have a minimum required size with the `min=""` attribute.
+- Objectives have a `required` attribute that specifies if they are required to complete the match.
+- A players gamemode can be changed by using the new `<game-mode> </game-mode>` kit.
+- New [world border module](/docs/modules/environment/border) that can be used to create a shifting and/or auto resizing world border.
+- Support added for block punch and trample `<block-drops>`.
+- Custom item [crafting recipe module](/docs/modules/gear/crafting).
+- All items of a specific type can be modified for the whole match using the [`<item-mods>` module](/docs/modules/gear/item-mods).
+- Teamless gamemodes with the [`<players/>` module](/docs/modules/format/players).
+- Heads can be given to players in kits with the [`<head>` element](/docs/modules/gear/items/#custom-heads).
+- Cause filters have several new possible values they can filter for.
+- Added the [`<relation>` filter](/docs/modules/mechanics/filters/#player-relation-filters) to check the relation between two players.
+- The world generator can be modified with the [terrain module](/docs/modules/environment/world#terrain). This module allows the seed, void/vanilla generator, and world folder location to be specified.
+- XML files can now contain conditional `<if>` and `<unless>` statements. These statements allow modules to be loaded on specific servers or during holiday events.
+- The maps display gamemode can be specified with the `<gamemode>` element.
+- Added a [shield kit module](/docs/modules/gear/kits/#shield-kit-removable) that gives a player an absorption hearts recharging shield.
+- [Pickups module](/docs/modules/gear/pickups) for location specific kit pickups.
+  ***
+
+<span className="badge badge--info">Removed</span>
+<br />
+
+- The `<multitrade/>` tag has been removed, it is now always enabled.
+
+### Changes in 1.3.6
+
+<span className="badge badge--danger">Breaking</span>
+<br />
+
+- Regions for cores, destroyables, wools, portals & score boxes must be defined in a `<region>` sub-element or referenced in a `region` attribute.
+
+  - ```xml
+    <core>
+        <region>
+            <cuboid min="..." max="..."/>
+        </region>
+    </core>
+    <core region="my-region"/>
+    ```
+  - Similarly spawn regions must be defined in a `<regions>` sub-element or referenced in a `regions` attribute.
+  - ```xml
+    <spawns>
+        <spawn team="red">
+            <regions yaw="90">
+                <cuboid min="1,0,2" max="3,0,4"/>
+                <cylinder base="7,8,9" radius="5" height="0"/>
+            </regions>
+        </spawn>
+    </spawns>
+    ```
+
+- Everything scores 0 points by default, this means you have to explicitly define `<kills>` and `<deaths>` in the score module if you want players to get points for killing players.
+- The `<playable>` region module has been removed, this should now be handled using filters.
```

Comment:
`<playable>` and `<lanes>` are deprecated but currently present in PGM and not restricted by proto.

Reply:
Where are you seeing `<lanes>`? I wasn't able to find any reference of it in the original .mdx file.

Reply:
https://github.com/PGMDev/PGM/blob/dev/core/src/main/java/tc/oc/pgm/regions/RegionModule.java
`<playable>` and `<lanes>` do pretty much the same thing, so I used them interchangeably, my mistake. This line should also be deleted.